README.md

```diff
@@ -30,15 +31,15 @@ npm install -g
 
 ## Initialize MinIO Client
 
-You need five items in order to connect to MinIO object storage server.
+The following parameters are needed to connect to a MinIO object storage server:
 
-| Params    | Description                                                                                         |
-| :-------- | :-------------------------------------------------------------------------------------------------- |
-| endPoint  | URL to object storage service.                                                                      |
-| port      | TCP/IP port number. This input is optional. Default value set to `80` for HTTP and `443` for HTTPs. |
-| accessKey | Access key is like user ID that uniquely identifies your account.                                   |
-| secretKey | Secret key is the password to your account.                                                         |
-| useSSL    | Set this value to 'true' to enable secure (HTTPS) access                                            |
+| Parameter   | Description                                                                  |
+| :---------- | :--------------------------------------------------------------------------- |
+| `endPoint`  | URL for the object storage service.                                          |
```

Comment:
`endPoint` is not a URL, it's host of URL

Reply:
Fixed